GiniVision/Classes/CameraContainerViewController.swift

```diff
@@ -165,5 +168,29 @@ internal class CameraContainerViewController: UIViewController, ContainerViewCon
         
         self.contentController.present(alertViewController, animated: true, completion: nil)
     }
-
+    
+    fileprivate func showPhotoLibraryPermissionDeniedError() {
```

Comment:
We should show this alert in the `CameraViewController` ourselves so that it is shown using the Component API, too. Clients can configure the message , but we take care of the permission alert inside the library => no public API changes required.